test/e2e/framework/providers/gce/util.go

```diff
@@ -36,12 +36,12 @@ func RecreateNodes(c clientset.Interface, nodes []v1.Node) error {
 	for i := range nodes {
 		node := &nodes[i]
 
-		if zone, ok := node.Labels[v1.LabelZoneFailureDomain]; ok {
+		if zone, ok := node.Labels[v1.LabelTopologyZone]; ok {
```

Comment:
This should be beta label since line 44 also checks for LabelTopologyZone

Reply:
will remove